packages/bun-types/deprecated.d.ts

```
@@ -98,6 +98,42 @@ declare module "bun" {
     ): void;
   }
 
+  /**
+   * @deprecated Use {@link Serve.Options Bun.Serve.Options<T, R>} instead
+   */
+  type ServeOptions<T = undefined, R extends string = never> = Serve.Options<T, R>;
+
+  /** @deprecated This type is unused in Bun's types and might be removed in the near future */
+  type Platform =
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Platform'. Consider to delete it or rename it.

'Platform' is defined here:


(lint/suspicious/noRedeclare)